Answers.md

```diff
@@ -1,5 +1,21 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+
+Client: HTML is created in browser by JS. Less data is processed. Faster. Harder to set up. SEOs are less optimized since the go-to has always been server-side.
+
+Server: 
+When HTML is created on server and sent to client.
+Requests that data that is needed. Good for Search Engine Optimizations. I am part of a company that spends tens, thousands of dollars in SEO Optimizations/AdWords campaigns. Because of this, we have found that using a server-side routing, written in a dynamic server-side language to be the most optimal and beneficial. With all the good parts, comes the bad parts-server side routing also has its cons. A lot of unnecessary data is requested and can take a toll speed-wise and performance-wise. There are ways to improve them but it won’t be as quick as client-side.
```

Comment:
That's so interesting that you experienced how beneficial it was to have serve side routing on your own! 🔥